test/ilpQuoterSpec.js

```diff
@@ -0,0 +1,129 @@
+'use strict'
+
+const nock = require('nock')
+nock.enableNetConnect(['localhost'])
```

Comment:
I actually do not know. I saw that at other locations in the code and just included that change as I thought it must have some usefulness. See for instance: https://github.com/interledger/five-bells-notary/blob/master/test/healthSpec.js#L4